### PR TITLE
Added Response API and books 

### PR DESCRIPTION
**Pull request by TealNerd**
Basically CivGuide but as an api integrated into CivMenu. @rourke750 wanted me to do it this way so, uh, here ya go!

### CODE REVIEW

File: src/vg/civcraft/mc/civmenu/datamanager/DismissalDAO.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import org.bukkit.Bukkit;
+
+import vg.civcraft.mc.civmenu.CivMenu;
+import vg.civcraft.mc.civmenu.database.Database;
+import vg.civcraft.mc.civmodcore.Config;
+import vg.civcraft.mc.civmodcore.annotations.CivConfig;
+import vg.civcraft.mc.civmodcore.annotations.CivConfigType;
+import vg.civcraft.mc.civmodcore.annotations.CivConfigs;
+
+public class DismissalDAO {
```
Comment (rourke750) [inline src/vg/civcraft/mc/civmenu/datamanager/DismissalDAO.java:20]: Why are we creating a new db class?

Comment (TealNerd) [inline src/vg/civcraft/mc/civmenu/datamanager/DismissalDAO.java:20]: it's a dao to interact with the database

**Comment by Maxopoly** [comment]
Should this be merged?

**Comment by TealNerd** [comment]
I sure think so

**Comment by Maxopoly** [comment]
For some reason I dont have permission to merge stuff here, @ttk2 @rourke750  could one of you merge this?

**Comment by erocs** [comment]
Try now.

**Comment by Maxopoly** [comment]
Thanks

**Comment by TealNerd** [comment]
WOHOOOO. Now just gotta integrate it into all the other plugins

**Pull request merged**